blocks/init/src/blocks/custom/video/manifest.json

```diff
@@ -12,9 +12,11 @@
     "preview"
   ],
   "example": {
-    "media": {
-      "id": 0,
-      "url": "https://storage.googleapis.com/coverr-main/mp4%2FIn-The-Clouds.mp4"
+    "attributes": {
+      "media": {
+        "id": 0,
+        "url": "https://storage.googleapis.com/coverr-main/mp4%2FIn-The-Clouds.mp4"
```

Comment:
Whose video is this, and what is the license of it? Can we use it in our project?

Reply:
I have used a placeholder video from this list

https://gist.github.com/jsturgis/3b19447b304616f18657

Reply:
Couldn't find it on that list. We should check the license and add attribution if reuse is permitted.

Reply:
I will open an issue